### DISCUSSION TITLE
Pre-populate the note with custom metadata

### DISCUSSION THREAD

**Discussion by vHanda** [GitJournal/GitJournal#746] [Feature Requests]
Right now you can add some custom metadata per note, but it's cumbersome and difficult to customize. It would be nice if there could be some kind of templating language involved and if we could support Dendron's deafults as an option.

https://github.com/dendronhq/dendron/blob/master/packages/common-all/src/nodev2.ts#L38:L38

---
> Please vote on this issue with a 👍 reaction. It's one of the primary mechanisms used to prioritize issues. Please also subscribe to the issue if you're interested, as that way I can discuss possible implementations and share mockups.

**Comment by bisraelsen** [comment]
As an example, I'd be interested in storing lat/long coordinates, and the associated weather conditions.
